vnext/Microsoft.ReactNative/Views/TextInputViewManager.cpp

```diff
@@ -513,6 +513,13 @@ void TextInputShadowNode::updateProperties(winrt::Microsoft::ReactNative::JSValu
         m_hideCaret = propertyValue.AsBoolean();
         HideCaretIfNeeded();
       }
+    } else if (propertyName == "focusable") {
+      // parent class also sets isTabStop
+      if (propertyValue.Type() == winrt::Microsoft::ReactNative::JSValueType::Boolean) {
+        control.IsEnabled(propertyValue.AsBoolean());
+      } else if (propertyValue.IsNull()) {
+        control.ClearValue(xaml::Controls::Control::IsEnabledProperty());
+      }
```

Comment:
I don't think we want to disable the whole control if it is not focusable.

Do you know if we wire `editable` to `IsEnabled`?

Reply:
`Editable` is actually wired to `isReadOnly` for a plain TextInput. Currently there is no way to access `isEnabled` through a plain TextBox. 

Setting `isEnabled` for `focusable` just changes the visual since you can't interact currently with a Textbox that has `focusable = false`. The Microsoft Docs for [isTabStop](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.control.istabstop?view=winrt-22000#windows-ui-xaml-controls-control-istabstop) also suggests setting `isEnabled` when ` isTabStop` is set.

Reply:
This doesn't really feel right to me, in terms of behavior. Is this consistent with other platforms?

cc @acoates-ms in case he knows the answer off the top of his head.

Reply:
A really simple way to check "reference behavior" would be to use Expo Snack. https://snack.expo.dev/

Android also has first class support for "focusable" (not sure about iOS). Are inputs that are not focusable disabled on Android?

Reply:
It looks like you can edit a TextInput that has `focusable = false` on Android and iOS [(snack example),](https://snack.expo.dev/@tatianak/textinput) but currently RNW makes it so you can't edit/paste/copy from a TextInput. This PR doesn't change the functionality just visually shows what RNW is doing.

Perhaps this is a larger issue of 1. We should enable editing a TextInput with `focusable = false` (although not sure how to do that when we set `isTabStop` for focusable), and 2. Exposing` isEnabled` through a different property of TextInput.